node/cli/src/command.rs

```diff
@@ -778,19 +778,19 @@ pub fn run() -> Result<()> {
 					#[cfg(feature = "moonriver-native")]
 					spec if spec.is_moonriver() => {
 						let block: service::moonriver_runtime::Block =
-							generate_genesis_block(&spec, state_version)?;
+							generate_genesis_block(&**spec, state_version)?;
```

Comment:
what the heck does ** mean?

Reply:
ahh, this is from before we chatted about just looking at the parachain staking. although i'm still curious if you know why